src/main/java/uk/gov/hmcts/reform/slc/services/steps/sftpupload/FtpUploader.java

```diff
@@ -53,12 +52,19 @@ public void upload(PdfDoc pdfDoc) {
         Instant start = null;
 
         try {
-            SFTPFileTransfer fileTransfer = getSftpFileTransfer();
+
             start = Instant.now();
 
-            fileTransfer.upload(pdfDoc, pdfDoc.filename);
+            ssh.addHostKeyVerifier(fingerprint);
+            ssh.connect(hostname, port);
+            ssh.authPassword(username, password);
+
+            try (SFTPClient sftp = ssh.newSFTPClient()) {
+                sftp.getFileTransfer().upload(pdfDoc, pdfDoc.filename);
+            }
```

Comment:
now it's not really tracking whether the upload failed. can it be changed to
```java
try (SFTPClient sftp = ssh.newSFTPClient()) {
    SFTPFileTransfer fileTransfer = sftp.getFileTransfer();
    start = Instant.now();
    fileTransfer.upload(pdfDoc, pdfDoc.filename);
    insights.trackFtpUpload(Duration.between(start, Instant.now()), true);
}
```

Reply:
it does, in line 66

Reply:
the start instant is assigned right at the beginning of the block - it will always be not null whenever the IOException is caught: getting new sftp client, getting file transfer or trying to upload. insight metric should be interested in the latter process. the rest - track exception

Reply:
or any previous method calls of `ssh` var